html/openstef_dashboard_doc.html

```diff
@@ -0,0 +1,53 @@
+<!--
+SPDX-FileCopyrightText: 2017-2021 Contributors to the OpenSTF project <korte.termijn.prognoses@alliander.com>
+
+SPDX-License-Identifier: MPL-2.0
+-->
+<!DOCTYPE html>
+<html class="writer-html5" lang="en" >
+
+<head>
+    <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+<title>Description of OpenSTEF dashboard</title>
```

Comment:
Misschien kan je op deze pagina nog een korte instructie toevoegen dat je een uitleg kunt zien door over verschillende onderdelen te muizen? Bijv in een pop-up als je de pagina opent, of gewoon door tekst aan het plaatje toe te voegen